http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/json/BookIdWrapper.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.ts.http.restclient.reactive.json;
+
+public class BookIdWrapper {
+
+    private final int id;
+
+    public BookIdWrapper(int id) {
+        this.id = id;
+    }
+
+    public BookIdWrapper(String id) {
```

Comment:
Is this constructor used anywhere?

Reply:
Yep, if you get rid of it, build fails
```
Caused by: java.lang.RuntimeException: Unable to create param converter for parameter class io.quarkus.ts.http.restclient.reactive.json.BookIdWrapper
```
as resource method parameters must comply one of following conditions:
- Be a primitive type
- Have a constructor that accepts a single String argument
- Have a static method named valueOf() or fromString() that accepts a single String argument, for example, Integer.valueOf(String)
- Have a registered implementation of ParamConverterProvider JAX-RS extension SPI that returns a ParamConverter instance capable of converting a string to a particular type
- Be List<T>, Set<T>, or SortedSet<T>, where T satisfies 2, 3, or 4 above. The resulting collection is read-only


I opted for constructor as it's not really relevant for this test case. Anyway, this comment reminded me I didn't actually used param convertor, so thanks for a hint!